src/publicapi.js

```diff
@@ -69,12 +69,21 @@ var AbstractMathQuill = P(function(_) {
   };
   _.config =
   MathQuill.config = function(opts) {
-    for (var opt in opts) if (opts.hasOwnProperty(opt)) {
+    for (var opt in opts) if (opts.hasOwnProperty(opt) && opts[opt] != null) {
```

Comment:
**caution** Are you sure that falsey values should be filtered out here? I would suggest not _enhancing_ anything here.